.spelling

```diff
@@ -0,0 +1,26 @@
+# markdown-spellcheck spelling configuration file
+# Format - lines beginning # are comments
+# global dictionary is at the start, file overrides afterwards
+# one word per line, to define a file override use ' - filename'
+# where filename is relative to this configuration file
+stateful
+webhooks
+webhook
+websocket
+ - blog/2024-09-13-first-documenation-iteration.md
+Fishjam
```

Comment:
Should this be in global override?

Reply:
i moved everything to global scope as the words will probably repeat in different scopes